herbieapp/management/commands/init_pubsub.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from django.core.management import BaseCommand
+
+from herbieapp.services import SchemaPackage
+from herbieapp.services.message_publisher.google_pub_sub_publisher import GooglePubSubPublisher
+
+
+class Command(BaseCommand):
+    help = 'initialize pubsub topics'
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._logger = logging.getLogger(__name__)
+        self._publisher = GooglePubSubPublisher()
+        self._schema_package = SchemaPackage()
+
+    def handle(self, *args, **kwargs):
+        names = self._schema_package.get_all_schema_names()
+        print(names)
+        for name in names:
+            self._publisher.create_topic(name)
```

Comment:
I am currently thinking if we should remove this command and create the topic in case it is not there as part of the publishing process. What do you think?

Reply:
Good idea :+1:

Reply:
Jira Ticket Created for this